new-examples/jdbc-execute-operation/jdbc_execute_operation.metatags

```diff
@@ -0,0 +1,2 @@
+description: BBE on the use of JDBC client with DDL and DML operations in Ballerina.
+keywords: ballerina, ballerina by example, bbe, jdbc, insert, update, delete
```

Comment:
```suggestion
keywords: ballerina, ballerina by example, bbe, jdbc, insert, update, delete
```
Please change this to match each of the BBEs.

Reply:
@praneesha Not clear about the suggestion ?